python_coreml_stable_diffusion/torch2coreml.py

```diff
@@ -1392,6 +1425,13 @@ def parser_spec():
         "If specified, enable unet to receive additional inputs from controlnet. "
         "Each input added to corresponding resnet output."
         )
+    parser.add_argument(
```

Comment:
Can we rename this to `--unet-compute-precision-float32`? Sidebar: I still don't understand why the float32 constraint would only apply to the CLI and not any other Swift program that calls `StableDiffusionXLPipeline` 🤔 We should discuss this more 🙏

Reply:
Yea I'll rename it. The CLI thing is still a mystery to me, I'd love to hear if anyone can replicate it or if it's just my machine. I was just getting noise on the output with using fp16, which went away when in fp32 format.

Reply:
I am still traveling with bad internet so I am yet to convert and test the refiner model.. (hence the slow review). Will confirm once I do!